### ISSUE TITLE
No data available for linux .tar and .rpm filebeat under "New users and groups ECS" dashboard.

### ISSUE THREAD

**Issue by amolnater-qasource** [elastic/beats#24315]
**Kibana version:** 7.12.0 BC-3 Kibana Cloud environment

**Host OS and Browser version:** Ubuntu 20(.tar), Centos 8.0(.rpm), All

**Build Details:**
```
Artifact link: https://staging.elastic.co/7.12.0-96914cb5/summary-7.12.0.html
Build: 39134
Commit: 08417cbd6c15e4c866651a7dcdfeded58845206d
```

**Preconditions:**
1. 7.12.0 BC-3 Kibana cloud environment should be available.
2. Install Filebeat with .tar and .rpm packages.

**Steps to reproduce:**
1. Login to Kibana Cloud environment.
2. Navigate to Dashboard>"[Filebeat]New users and groups ECS".
3. Observe no data under "[Filebeat]New users and groups ECS".

**Expected Result:**
Data should be available under "New users and groups ECS" dashboard for linux .tar and .rpm filebeat.

**Screenshot:**
![3](https://user-images.githubusercontent.com/77374876/109798866-46f34a80-7c41-11eb-910f-7c62eb0c7d00.JPG)

**Test case reference:**
[View data in '[Filebeat System] New users and groups ECS' dashboard](https://elastic.testrail.io/index.php?/cases/view/15813&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=3504)

**What's working:**
Data is displayed for Linux and RPM under:
-  Discover tab.
- "[Filebeat System] SSH login attempts ECS" dashboard.
- "[Filebeat System] Syslog dashboard ECS" dashboard.
- "[Filebeat System] Sudo commands ECS" dashboard.

**Working Screenshot:**
![dashboard](https://user-images.githubusercontent.com/77374876/109808791-8f186a00-7c4d-11eb-92b2-317c4e58549f.png)

**Comment by amolnater-qasource** [comment]
@manishgupta-qasource Please review.

**Comment by manishgupta-qasource** [comment]
Reviewed & assigned to @EricDavisX

**Comment by EricDavisX** [comment]
pinging team, it sounds severe - will raise it to slack.

**Comment by fearful-symmetry** [comment]
Sorry, I'm not a filebeat expert, can we verify that there's been new user or group events that would populate the dashboard?

**Comment by EricDavisX** [comment]
@amolnater-qasource @dikshachauhan-qasource can you confirm / re-test... and while testing make sure that we are generating user / group data on the host to ensure it will / should be populated in the Dashboard?  If no data is sent, it isn't a bug - and we just need to update our test content to ensure proper steps.

**Comment by amolnater-qasource** [comment]
Hi @EricDavisX 
Thanks for the feedback. We have used following commands to create users and groups:
```
sudo useradd -m USER -p PASS
sudo groupadd GROUP_NAME
sudo usermod -a -G GROUP_NAME  USER
```
Further to check members:
```
sudo apt install members
members GROUP_NAME
```
With these commands we are able to view data under "New users and groups ECS" dashboard. 
**Screenshot:**
![1](https://user-images.githubusercontent.com/77374876/110076273-3ad6cc80-7daa-11eb-957d-5be37a9cd767.JPG)

We have updated our test case as per requirement: [View data in '[Filebeat System] New users and groups ECS' dashboard](https://elastic.testrail.io/index.php?/cases/view/15813)
Hence closing this out.

Thanks
QAS
